src/atoms/Link/Link.stories.js

```diff
@@ -18,4 +18,14 @@ storiesOf('Link', module)
     <Link size={select('Size', sizeOptions, 'large')}>
       {text('Display Text', 'https://www.razorpay.com')}
     </Link>
+  ))
+  .add('disabled', () => (
```

Comment:
Link doesn't have disabled and visited state on Figma. Why are we doing it here then?